content/about.en.md

```diff
@@ -12,30 +12,35 @@ As contributors to a common, we collectively do our best to make our community a
 
 ## History
 
-The development of OpenFisca began in May 2011 at the French [_Centre d'analyse stratégique_](https://strategie.gouv.fr/) (renamed _France Stratégie_ in April 2013), with the support of the [<abbr title="Institut d'économie publique">IDEP</abbr>](https://www.idep-fr.org/).
+The development of OpenFisca began in May 2011 at the French [_Centre d'analyse stratégique_](https://www.strategie.gouv.fr/) (renamed _France Stratégie_ in April 2013), with the support of the _Institut d'économie publique (IDEP, Aix-Marseille)_.
 
 OpenFisca was originally developed as a desktop application using the [Qt](https://www.qt.io/) library with a Python API. This source code was released under a free software (AGPL) license in November 2011.
 
 In early 2014, the French Prime Minister task force for open data, [Etalab](https://www.etalab.gouv.fr/), started using OpenFisca and soon became a major contributor. After organising a [hackathon](https://www.eventbrite.fr/e/billets-hackathon-openfisca-10751826001) to give more visibility to the product, its team decided to:
 
-- stop the development of the [Qt version](https://github.com/openfisca/openfisca-qt);
-- separate the computing engine from its desktop user interface;
-- offer a web API in addition to the Python API;
-- demonstrate the value of the web API by developing sample applications including a web interface to simulate personal cases;
-- offer public access to this web API.
+<ul>
+    <li>stop the development of the [Qt version](https://github.com/openfisca/openfisca-qt);</li>
+    <li>separate the computing engine from its desktop user interface;</li>
+    <li>offer a web API in addition to the Python API;</li>
+    <li>demonstrate the value of the web API by developing sample applications including a web interface to simulate personal cases;</li>
+    <li>offer public access to this web API.</li>
+</ul>
```

Comment:
@clementbiron This syntax doesn't fix the list. What would you recommend? ^^'

Reply:
This file is in Markdown and should stay this way 🙂 The issue is not with the HTML, it is that default list styles have been reset and they should be styled.